feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/SyncInfoViewModel.kt

```diff
@@ -195,15 +185,15 @@ internal class SyncInfoViewModel @Inject constructor(
         if (configRepository.getProjectConfiguration().isEventDownSyncAllowed()) {
             fetchAndUpdateRecordsToDownSyncAndDeleteCount()
         } else {
-            DownSyncCounts(0, 0)
+            DownSyncCounts(0, false)
         }
 
     private suspend fun fetchAndUpdateRecordsToDownSyncAndDeleteCount(): DownSyncCounts =
         try {
             eventSyncManager.countEventsToDownload()
         } catch (t: Throwable) {
             Simber.d(t)
-            DownSyncCounts(0, 0)
+            DownSyncCounts(0, false)
```

Comment:
In the future, I'd like us to use the explicit parameter names, especially in cases like this. `DownSyncCounts(0, false)` doesn't make much sense when reading. 

In comparison, `DownSyncCounts(count = 0, isLowerBound = false)` is easier to understand. Or at least specifying the flag's purpose: `DownSyncCounts(0, isLowerBound = false)`